debian/vyos-1x.postinst

```diff
@@ -122,5 +122,8 @@ if test -f /etc/pam.d/frr; then
     fi
 fi
 
+# Enable Cloud-init pre-configuration service
+systemctl enable vyos-config-cloud-init.service
```

Comment:
Is it a good idea to enable cloud-init service for every system by default?

Reply:
I had this feeling too, but it is safe.  :)

This command will install a symlink to the `cloud-init-local.service.wants` directory, which will be active only if the `cloud-init-local.service` is presented and enabled.